### ISSUE TITLE
How should I save the loss value during training? model.fit has no return value

### ISSUE THREAD

**Issue by Z-shuaishuai** [SciSharp/TensorFlow.NET#927]
How should I save the loss value during training?
model.fit has no return value

**Comment by Darlanio** [comment]
I think you need to use Tensorflow.Keras.Engine.Sequential instead of model,
and then use predict, calculate loss from the result of predict,
calculate new gradients with Tensorflow.Gradients.GradientTape,
and finally use your keras.optamizers.[chosen optimizer].apply_gradients().

However, I am less than sure about this. I hope Haiping can answer this - possibly correcting my answer.

**Comment by Oceania2018** [comment]
`model.fit` returns `History` for next release `v0.100.2`.
![image](https://user-images.githubusercontent.com/1705364/215641697-30db154b-29dc-4849-be1d-aa21fd9835a3.png)

![image](https://user-images.githubusercontent.com/1705364/215641643-9b656294-4479-45c8-b2b1-1b73ca2f87ab.png)

**Comment by ammar-albush** [comment]
When would the new version(v0.100.2) be available on **Nuget** ?

**Comment by Darlanio** [comment]
My guess, it will take at least a few days, probably a few weeks:
There are many small bugs to fix at present (927, 928... 973...)
